test/quantization/fx/test_equalize_fx.py

```diff
@@ -0,0 +1,111 @@
+import torch
+from torch.testing._internal.common_quantization import QuantizationTestCase
+from torch.quantization.fx._equalize import (
+    _InputEqualObserver, _WeightEqualObserver, calculate_equalization_scale
+)
+
+# Standard Libraries
+import numpy as np
+
+# Testing utils
+from hypothesis import given
+from hypothesis import strategies as st
+
+
+class TestEqualizeFx(QuantizationTestCase):
+    @given(input_qdtype=st.sampled_from((torch.qint8, torch.quint8)),
+           input_qscheme=st.sampled_from((torch.per_tensor_affine, torch.per_tensor_symmetric)),
+           weight_qdtype=st.sampled_from((torch.qint8, torch.quint8)),
+           weight_qscheme=st.sampled_from((torch.per_channel_affine, torch.per_channel_symmetric,
+                                           torch.per_channel_affine_float_qparams)))
+    def test_input_weight_observer(self, input_qdtype, input_qscheme, weight_qdtype, weight_qscheme):
+        input_obs = _InputEqualObserver(dtype=input_qdtype,
+                                        qscheme=input_qscheme)
+        weight_obs = _WeightEqualObserver(dtype=weight_qdtype,
+                                          qscheme=weight_qscheme)
+
+        width = np.random.randint(10) + 1
+        x_height = np.random.randint(10) + 2
+        w_height = np.random.randint(10) + 2
```

Comment:
Why do we add the constant values here?

Reply:
The constant values are to make sure that the width is greater than 0 and the heights are greater than 1. If the heights are equal to 1 then the min/max values will be the same and there will be a division by 0 error when calculating the scale.

Reply:
I can also replace them with `w = np.random.randint(1, 10)` and `h = np.random.randint(2, 10)` if that's more readable.